pennylane/qchem/structure.py

```diff
@@ -282,7 +282,32 @@ def excitations(electrons, orbitals, delta_sz=0):
     return singles, doubles
 
 
-def hf_state(electrons, orbitals):
+def _beta_matrix(orbitals):
+    r"""Generate the beta matrix for conversion from occupation number basis to the Bravyi-Kitaev basis.
+
+    Args:
+        orbitals (int): Number of *spin* orbitals. If an active space is defined,
+            this is the number of active spin-orbitals.
+
+    Returns:
+        (array): Transformation matrix beta
+    """
+
+    # Base case for beta_1
+    if orbitals == 1:
+        return np.array([[1]])
+
+    bin_range = int(np.ceil(np.log2(orbitals)))
+
+    beta_previous = _beta_matrix(2 ** (bin_range - 1))
+    beta = np.kron(np.eye(2), beta_previous)
+    # Fill in the last row of the last quadrant with 1's
+    beta[-1, : 2 ** (bin_range - 1)] = 1
+
+    return beta[:orbitals, :orbitals]
+
+
+def hf_state(electrons, orbitals, basis="occupation_num"):
     r"""Generate the occupation-number vector representing the Hartree-Fock state.
```

Comment:
```suggestion
    r"""Generate the vector representing the Hartree-Fock state.
```

Reply:
Need to update/generalise the rest of the doctsring.

Reply:
```suggestion
    r"""Generate the Hartree-Fock statevector with respect to a chosen basis.
```
Another option.